packages/components/src/components/notice/bal-snackbar/bal-snackbar.tsx

```diff
@@ -131,9 +136,10 @@ export class Snackbar {
               </bal-heading>
             </span>
           </div>
-          <bal-text class="bal-snackbar__label" space={this.action === '' ? 'bottom' : 'none'} innerHTML={this.message}>
+          <span class="bal-snackbar__label" {...labelAttributes}>
             <slot />
```

Comment:
i think we need to use named slot in order for mangolia?
also will this not influence as well the inner html with this class?

Reply:
I tested with the beta version and it's working in Magnolia. So we don't need the named slot anymore

Reply:
perfect